src/modules/services/services.conf

```diff
@@ -1,20 +1,24 @@
 ---
-#systemd services and targets are enabled in this precise order
+# systemd services and targets example:
```

Comment:
Needs documentation on the meaning of services, targets, and disable. "There are three keys in this configuration, ... . Each holds a list of systemd units (?) to *something-something*. Use [] to explicitly denote an empty list (which is the default value for each key). .."